apps/dashboard_app/helpers/protocol_stats.py

```diff
@@ -116,7 +121,11 @@ def get_supply_stats(
     df = pd.DataFrame(data)
     df["Total supply (USD)"] = sum(
         df[column]
-        * Decimal(prices[TOKEN_SETTINGS[column.replace(" supply", "")].address])
+        * Decimal(
+            prices[
+                add_leading_zeros(TOKEN_SETTINGS[column.replace(" supply", "")].address)
```

Comment:
Didn't it effect other tokens? I remember that there was issue with 2 of tokens?

Reply:
I don’t see any errors on my database. The issue might be related to a specific dataset